src/lib.rs

```diff
@@ -1,8 +1,5 @@
 #![doc = include_str!("../README.md")]
 #![cfg_attr(feature = "nightly", feature(slice_take))]
-#![feature(async_closure)]
-#![feature(io_error_more)]
-#![feature(io_error_uncategorized)]
```

Comment:
Why drop the `io_error*` features?

Reply:
I'll admit that this is a little outside the scope of this PR. I did this because it was unclear if the nightly build was producing a static library that uniffi-bindgen couldn't consume (it was the enum constructor issues).

Anyway, I think it would be nice to get off the nightly compiler when the nightly feature flag isn't enabled. We're stuck with it for watchOS and tvOS but not for macOS, iOS and android targets.

If you feel strongly about this I can take it out of this PR.

Reply:
It's fine either way.  It's used only because the source library I was wrapping supported it too.  If it builds fine without it then it doesn't matter until one of the consumers in Swift or Kotlin needs more detailed errors.